Kotlin/app/src/main/java/realm/io/realmsurveys/SurveyApplication.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package realm.io.realmsurveys
+
+import android.app.Application
+
+import io.realm.Realm
+import io.realm.log.LogLevel
+import io.realm.log.RealmLog
+
+class SurveyApplication : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        Realm.init(this)
+        RealmLog.setLevel(LogLevel.TRACE)
```

Comment:
Is this really needed? It will massively flood their logs. The default is WARN, which is enough for most cases unless debugging